pkg/internal/utils.go

```diff
@@ -199,9 +212,16 @@ func ExtractPodBindInfo(allocatedPod *core.Pod) *si.PodBindInfo {
 }
 
 func ExtractPodBindAnnotations(allocatedPod *core.Pod) map[string]string {
-	return map[string]string{
-		si.AnnotationKeyPodGpuIsolation: allocatedPod.Annotations[si.AnnotationKeyPodGpuIsolation],
-		si.AnnotationKeyPodBindInfo:     allocatedPod.Annotations[si.AnnotationKeyPodBindInfo],
+	if _, ok := allocatedPod.Annotations[si.AnnotationKeyPodLeafCellIsolation]; ok {
+		return map[string]string{
+			si.AnnotationKeyPodLeafCellIsolation: allocatedPod.Annotations[si.AnnotationKeyPodLeafCellIsolation],
+			si.AnnotationKeyPodBindInfo:          allocatedPod.Annotations[si.AnnotationKeyPodBindInfo],
+		}
+	} else {
+		return map[string]string{
+			si.AnnotationKeyPodLeafCellIsolation: allocatedPod.Annotations[strings.Replace(si.AnnotationKeyPodLeafCellIsolation, "leaf-cell", "gpu", -1)],
```

Comment:
>strings.Replace(si.AnnotationKeyPodLeafCellIsolation, "leaf-cell", "gpu", -1) [](start = 66, length = 77)

Also define the deprecated Annotation in constants.go, instead of generating in runtime.
Such as DeprecatedAnnotationKeyXXX

Reply:
updated